docker/nginx/proxito.conf

```
@@ -20,10 +20,10 @@ server {
     }
 
     # Sendfile support to serve the actual files that Proxito has specified
-    location /proxito/ {
+    location /proxito/devstoreaccount1/media/ {
         internal;
-        # Nginx will strip the `/proxito/` and pass just the `html/$proj/$ver/$filename`
-        # So we need to make sure we pass it to `media/` for the proper URL
+        # Nginx will strip the `/proxito/devstoreaccount1/media/` and pass just the `html/$proj/$ver/$filename`
+        # So we need to make sure we pass it to `/devstoreaccount1/media/` for the proper URL
         proxy_pass http://storage:10000/devstoreaccount1/media/;
```

I think a better fix here is to just stop prepending it, and then any `/proxito/` URL will just point at the proper path on the storage server. 

```suggestion
         proxy_pass http://storage:10000/;
```